.github/workflows/notify-slack.yml

```diff
@@ -0,0 +1,33 @@
+name: Check pull request description checkbox and send notify to Slack
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
@AndriiHoropatskyiDelinea should this be closed?

Reply:
@tylerezimmerman it needs to be merged into main, for now it's final version